HW_1/task2.py

```diff
@@ -0,0 +1,34 @@
+# Задача 2
+
+# Треугольник существует только тогда, когда сумма любых двух его сторон больше третьей.
+# Дано a, b, c - стороны предполагаемого треугольника. Требуется сравнить длину каждого
+# отрезка-стороны с суммой двух других. Если хотя бы в одном случае отрезок окажется больше
+# суммы двух других, то треугольника с такими сторонами не существует. Отдельно сообщить является
+# ли треугольник разносторонним, равнобедренным или равносторонним.
+
+
+#Просим пользователя ввести длину стороны
+a = int(input("Введите длину первой стороны треугольника: "))
+b = int(input("Введите длину второй стороны треугольника: "))
+c = int(input("Введите длину третьей стороны треугольника: "))
+
+
+def triangle_type():
+
+    if a + b > c and a + c > b and b + c > a:
+        # Треугольник существует
+        if a == b == c:
+            return "равносторонний"
+        elif a == b or a == c or b == c:
+            return "равнобедренный"
+        else:
+            return "разносторонний"
+    else:
+        # Треугольник не существует
+        return "не существует"
```

Comment:
Чуток подправлю, а вы внимательно посмотрите, что я сделал
```suggestion
        if a == b == c:
            return "равносторонний"
            
        if a != b != c:
            return "разносторонний"
            
        return "равнобедренный"

    # Треугольник не существует
    return "не существует"
```